Jint/Native/Function/ScriptFunctionInstance.cs

```diff
@@ -94,6 +95,16 @@ public override JsValue Call(JsValue thisArg, JsValue[] arguments)
 
                     if (result.Type == Completion.Throw)
                     {
+                        if (result.Value.HasValue && result.Value.Value.IsObject())
+                        {
+                            var e = result.Value.Value.ToObject() as Exception;
+                            if (e != null)
+                            {
+                                e.Rethrow();
```

Comment:
Simple `throw e; ` will replace stack trace. `Rethrow` method tries to preserve it.